CHANGELOG.md

```diff
@@ -4,6 +4,25 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.18.0
+
+This release adds the `beforeSendTransaction` callback to all JS SDKs, letting you make changes or drop transactions before they're sent to Sentry. This callback works identically to `beforeSend`, just for transactions.
+
+Furthermore, this release adds the `shouldCreateSpanForRequest` option to the Node SDK and the `tracePropagationTargets` option to the Browser SDK, providing fine grained control over which outgoing Http requests should be represented as a span in an ongoing transaction and which requests should be traced by Sentry.
+
+This release also marks the starting point of adding support for OpenTelemetry to the Node SDK, by adding the `instrumenter` option to the Node SDK's init options.
```

Comment:
h: I would rather not include this note yet, and remove the otel related changelog item until we make the package public.

Reply:
Done [522518e](https://github.com/getsentry/sentry-javascript/pull/6151/commits/522518e3667b038a56855c0c3b98459a9ee8963a)